src/Reader/XLSX/Sheet.php

```diff
@@ -47,6 +48,14 @@ public function getRowIterator(): RowIterator
         return $this->rowIterator;
     }
 
+    /**
+     * @return ColumnWidth[] a list of column-widths
+     */
+    public function getColumnWidths(): array
```

Comment:
Is this new API really needed? I mean, what's the use case of iterating column widths WITHOUT iterator rows first?

Reply:
It's strictly not necessary, but it came from a more semantic point that I think the 'column widths' aspect is something that should is part of a sheet, not specifically of a rowiterator. 

In the current setup of the code, all the XML parsing of the sheet is done in the RowIterator, which makes sense if you are only concerned with the row data. But the sheet-metadata (such as column widths) is now also being parsed by the RowIterator, ideally I think that should happen somewhere else: on sheet level. Because now you first have to `rewind()` the RowIterator to make it read the header of the sheet, to be able to retrieve the column widths. 

That's the reason I put the PR back into draft, because I'm not too happy with this, but refactoring the XML parsing into 2 parts (one bit for reading the header of the sheet and one for iterating over the rows) seemed like quite a bit of work to achieve something quite simple as getting the column sizes. 

So by introducing the method on Sheet, it should be possible to do this refactorment later, without breaking the API. That also means that the `RowIterator::getColumWidths()` should be marked `@internal` I guess.